cfme/common/physical_server_views.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+from lxml.html import document_fromstring
+from widgetastic.utils import (
+    Parameter,
+    Version,
+    VersionPick
+)
+from widgetastic.widget import ParametrizedView, Text, View
+from widgetastic_manageiq import (
+    BaseEntitiesView,
+    NonJSBaseEntity,
+    JSBaseEntity,
+    BaseListEntity,
+    BaseQuadIconEntity,
+    BaseTileIconEntity,
+    BootstrapTreeview,
+    BreadCrumb,
+    Button,
+    ItemsToolBarViewSelector,
+    PaginationPane,
+    SummaryTable,
+    TimelinesView,
+    BaseNonInteractiveEntitiesView
+)
+from widgetastic_patternfly import (
+    Dropdown,
+    FlashMessages,
+)
+
+from cfme.base.login import BaseLoggedInPage
+
+
+class ComputePhysicalInfrastructureServersView(BaseLoggedInPage):
+    """Common parts for server views."""
+    title = Text('.//div[@id="center_div" or @id="main-content"]//h1')
+    flash = FlashMessages(
```

Comment:
flash is included in BaseLoggedInPage, and doesn't need to be redefined here.  Don't have to change any calls for it, widgets in inherited views are 'flat'.